### PR TITLE
[DOCS] Add CONTRIBUTING GUIDELINES #4

### PR DESCRIPTION
**Pull request by frazie**
contributing guidelines to tap-for-a-surprise

### CODE REVIEW

**Comment by frazie** [comment]
In case of any issues, I'll be available to assist
I'm also open to any feedback. Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:11]:
```suggestion
We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
```

capitalize as required

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Contributing to tap-for-a-surprise
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:10]:
```suggestion
## We Develop with GitHub
```

capitalize as required

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:13]:
```suggestion
## We Use [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
```

capitalize as required

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow). We actively welcome your pull requests:
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:14]:
```suggestion
Pull requests are the best way to propose changes to the codebase (we use [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow). We actively welcome your pull requests:
```

capitalize as required

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues]()
```
Comment (KendallDoesCoding) [inline CONTRIBUTING.md:26]:
```suggestion
## Report bugs using GitHub's [issues]()
```

capitalize as required

**Comment by KendallDoesCoding** [comment]
I hope the effect comes to the same thing if I move it to the .github directory, please confirm and will merge accordingly @frazie

**Comment by frazie** [comment]
Confirmed
You can proceed

**Comment by KendallDoesCoding** [comment]
> Confirmed @KendallDoesCoding You can proceed

Thanks

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @frazie !

**Pull request merged**